kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/pom.xml

```
@@ -791,6 +817,34 @@
               </target>
             </configuration>
           </execution>
+          <execution>
+            <id>Copy kogito-editors-js assets to WAR</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <copy todir="${basedir}/src/main/webapp/kogito-editors-js">
+                  <fileset dir="${project.build.directory}/kogito-editors-js/META-INF/resources/webjars/kogito-editors-js/kogito-editors-js" includes="**/*" />
+                </copy>
+              </target>
+            </configuration>
+          </execution>
+          <execution>
+            <id>Remove unused appformer-js-monaco assets</id>
+            <phase>install</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <tasks>
+                <delete>
```

@vpellegrino I was able to reproduce the build failure locally. It happens if we skip GWT compilation. We may add this change to solve it:

```suggestion
                <delete quiet="true">
```